packages/core/src/cli/config.ts

```diff
@@ -115,6 +116,7 @@ export async function loadConfigByPath(configFile: string) {
       const params: ConfigParams = {
         env: getNodeEnv(),
         command,
+        envMode,
```

Comment:
should we use `getNodeEnv()` as the default value of `envMode`?

Adding default values can be consistent with the behavior of `loadEnv` method